ui/index.ts

```diff
@@ -68,7 +80,6 @@ export {
   OAuthCallback,
   RepoInputWithAuth,
   SignIn,
```

Comment:
Shall we bring over the `Page` wrapper and use that in EE too?

Reply:
I can do that - do you want to use it everywhere in enterprise? Only asking as `Page` includes stuff (title bar, loading screen) I'd guess you'd want to keep consistent in enterprise between core bits and other bits?

Reply:
Yeah, maybe to play with? I'm still not sure of what the ideal long term API of this should be. It sounds convenient and consistent to export a layout and page system that we can embrace in EE too. But sometimes you wanna bust out..

Reply:
I've exported the `Page` wrapper in a sneaky post-approval-pre-merge commit.